### DISCUSSION TITLE
Method to share all feeds from a label or category

### DISCUSSION THREAD

**Discussion by yochaigal** [FreshRSS/FreshRSS#6193] [Q&A] [answered]
Coming from Inoreader, where I could share a live OPML of all my feeds from one folder, I would love to be able to do the same thing here. That way I should share a URL with folks that want to import an up-to-date version of all my feeds (I maintain a pretty big list). I'd love to be able to click on a category or label and share out a permanent URL with all the feeds.

**Comment by Alkarex** [comment, marked as answer]
FreshRSS natively supports resharing of any view via RSS, OPML, and (since FreshRSS 1.24) HTML.
The method has changed a bit since FreshRSS 1.24 (use `edge` version to get the new behaviour today).
https://freshrss.github.io/FreshRSS/en/users/user_queries.html#share-your-user-queries

**Comment by yochaigal** [comment]
Thanks, that's great! 
I'd had a lot of trouble upgrading to the edge branch (by way of git and by way of the zip) so I sort of gave up. I'll wait for that to become the stable build and update through the regular channel.

**Reply by Alkarex** [reply]
Updating to the [`edge` branch](https://github.com/FreshRSS/FreshRSS/tree/edge) should be very straightforward. If you tell how you installed your current FreshRSS, we might be able to guide some more

**Reply by yochaigal** [reply]
Thanks - I am familiar with it (I'm a linux sysadmin by trade)! Let me know if an issue is better.

I pulled both the latest repo for that branch and copied it over to the directory (/usr/share/webapps/freshrss for me), preserving both "p" and "data", but am prompted to then set up FreshRSS from scratch! 

My config is:
- Arch linux
- latest zip release
- installed to /usr/share/webapps/freshrss, with a symlink to /srv/nging/freshrss
- Using local port for php-fpm (other services rely on it)

My nginx reverse proxy:
```	location ~ ^.+?\.php(/.*)?$ {
	fastcgi_pass 127.0.0.1:9000; 
	fastcgi_split_path_info ^(.+\.php)(/.*)$;
	# By default, the variable PATH_INFO is not set under PHP-FPM
	# But FreshRSS API greader.php need it. If you have a “Bad Request” error, double check this var!
	# NOTE: the separate $path_info variable is required. For more details, see:
	# https://trac.nginx.org/nginx/ticket/321
	set $path_info $fastcgi_path_info;
	fastcgi_param PATH_INFO $path_info;
	include fastcgi_params;
	fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
	}
```

I do wonder if one issue is that I use an nginx reverse proxy pointing to a folder (e.g. my.domain.com/freshrss) instead of leaving it at the root web directory. I recall this coming up a lot when I first set FreshRSS up, but have forgotten whatever it was I did to fix it, haha.
